app/models/admin_dashboard_data.rb

```diff
@@ -125,7 +130,7 @@ def self.load_found_scheduled_check_problems
   end
 
   def self.register_default_scheduled_problem_checks
-    add_scheduled_problem_check(:group_smtp_credentials) do
+    add_scheduled_problem_check(:group_smtp_credentials, GroupEmailCredentialsCheck) do
```

Comment:
In the future clean-up this class name can be inferred from file location and check identifier but for now must pass explicitly.